doc/release-notes/0.45/0.45.md

```diff
@@ -0,0 +1,88 @@
+<!--
+* Copyright (c) 2024 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# Eclipse OpenJ9 version 0.45.0 release notes
+
+These release notes support the [Eclipse OpenJ9&trade; 0.45.0 release plan](https://projects.eclipse.org/projects/technology.openj9/releases/0.45.0/plan).
+
+## Supported environments
+
+OpenJ9 release 0.45.0 supports OpenJDK 22.
+
+All releases are tested against the OpenJ9 functional verification (FV) test suite, the OpenJDK test suites, and additional tests provided by Adoptium.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+## Notable changes in this release
+
+There are no changes from v0.44.0 in v0.45.0. Information about the changes in v0.44.0 can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.44/).
```

Comment:
This would be better worded as "There are no additional issues or features ...".
Otherwise it can be read as none of the changes in 0.44 are in 0.45. There are changes to support Java 22, just not ones we call out in What's New or these release notes.

Reply:
Done